src/pages/MessagesPage.vue

```diff
@@ -1,3 +1,32 @@
 <template>
-  <h1>Messages Page</h1>
+  <div class="messages-page">
+    <TheFloatingButton />
+    <AppModal modalTitle="Create Message">
+      <ModalCreateMessage />
+    </AppModal>
+    <MessagesDataTable />
+  </div>
 </template>
+
+<script>
+import TheFloatingButton from '@/components/TheFloatingButton.vue';
+import AppModal from '@/components/AppModal.vue';
+import ModalCreateMessage from '@/components/ModalCreateMessage.vue';
+import MessagesDataTable from '../components/MessagesDataTable.vue';
```

Comment:
Use one way to write this '@/' or '../' . We used '../' in our project.

Reply:
Actually you should always prefer relative imports so `@/`, I did not notice this earlier, so you can just change these to use relative imports for now and some time later update the others ( not in this PR )

Reply:
@josip2312 Fixed.